File: DemoMMOUnity/Lua/application/app/ui/generateScripts/UIModules/MapGuide11000View.lua
---@class MapGuide11000View
MapGuide11000View = class("MapGuide11000View")
function MapGuide11000View:Init(view)
	self._UIView = view:GetComponent("UIView")
	self.Grid01_Button = self._UIView:GetObjType("Grid01_Button") or UnityEngine.UI.Button
end

File: UnityMMOClient/Assets/Game/Hoftix/Script/GenerateScripts/UIModules/MapGuide11000View.cs
using ZJYFrameWork.UISerializable.UIInitView;
using UnityEngine;
using UnityEngine.UI;

namespace ZJYFrameWork.UISerializable
{
    public class MapGuide11000View:UIViewInterface
    {
        public UnityEngine.UI.Button Grid01_Button=null;
		


        public void Init(UIView _view)
        {
            Grid01_Button=_view.GetObjType<UnityEngine.UI.Button>("Grid01_Button");
			
        }
    }
}